daily-builds.md

```diff
@@ -1,11 +1,21 @@
-# .NET Core Daily Builds -- Preview Releases
+# .NET Core Daily Builds -- Servicing and future releases
 
-The .NET Core team builds several branches of the product daily. You can install these builds to try out pre-released functionality. The branches are typically either for preview or [servicing versions](daily-builds-servicing.md) of the product.
+The .NET Core team builds many branches of the product daily. You can install these builds to try out pre-released functionality.  These brnches
 
 Preview branches are for new major/minor versions of .NET Core that have not yet been released and are not yet supported.
```

Comment:
Move this block **under** the `## Future Releases` section and **above** the version table... i.e. move line 5 down to line 8 (and add a blank line)